composites/Plugin/Shared/components/Toggle.js

```diff
@@ -0,0 +1,152 @@
+import PropTypes from "prop-types";
+import React from "react";
+import styled from "styled-components";
+import colors from "../../../../style-guide/colors.json";
+import { __ } from "@wordpress/i18n";
+
+const ToggleBar = styled.div`
+	background-color: ${ props => props.isEnabled ? "#a5d6a7" : colors.$color_button_border };
+	border-radius: 7px;
+	height: 14px;
+	width: 30px;
+	cursor: pointer;
+	margin: 0;
+	outline: 0;
+	&:focus > span {
+		box-shadow: inset 0 0 0 1px ${colors.$color_white}, 0 0 0 1px #5b9dd9, 0 0 2px 1px rgba(30, 140, 190, .8);
+	}
+`;
+
+const ToggleBullet = styled.span`
+	background-color: ${ props => props.isEnabled ? colors.$color_green_medium_light : colors.$color_grey_medium_dark };
+	margin-left: ${ props => props.isEnabled ? "12px" : "-2px" };
+	box-shadow: 0 2px 2px 2px rgba(0, 0, 0, 0.1);
```

Comment:
Same here: this is a hardcoded color. We should use a color from the palette (not necessarily a rgba color)

Reply:
Hi Andrea, you are right. But we'll skip this for now to finish the component. We'll pick this up again during the polishing process.